spec/models/intervention_type_spec.rb

```diff
@@ -34,15 +34,15 @@
       intervention_type_1 = create(:intervention_type, name: 'foo')
       intervention_type_2 = create(:intervention_type, name: 'bar')
 
-      expect(InterventionType.search('foo')).to eq([intervention_type_1])
-      expect(InterventionType.search('bar')).to eq([intervention_type_2])
+      expect(InterventionType.search(query: 'foo', locale: 'en')).to eq([intervention_type_1])
```

Comment:
I think we should improve the tests to add both english and welsh attribute names and confirm that a search in English does not find Welsh results and vice versa. This should be fairly simple to add by setting the appropriate attributes and adding some variants of the searches.

The tests currently only assets that searching by name works, but can you also add tests that show that matching is being done on all of the indexed fields, e.g. name, summary and description. That could be done for a single locale.

Reply:
:+1: done